GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -3232,5 +3248,15 @@ private void FormBrowse_DragEnter(object sender, DragEventArgs e)
                 e.Effect = DragDropEffects.Move;
             }
         }
+
+        private void TsmiTelemetryEnabled_Click(object sender, EventArgs e)
+        {
+            UICommands.StartGeneralSettingsDialog(this);
```

Comment:
I'd prefer what I guess other user would expect, too. The only little drawback of this is that the link to the PrivacyPolicy.md is not directly available.
```suggestion
            AppSettings.TelemetryEnabled = !tsmiTelemetryEnabled.Checked;
```
BTW, the time it takes to open the settings dialog is really annoying.

Reply:
I specifically don't want to do that - I want to make users aware of why we collect telemetry and we can't show the link in the menus.

Agree, the settings is a little slow (it is slower in a Debug build), but I don't expect users constantly go back and forth... So delay is probably not an end of the world here. Besides it gives user a second chance to think 😉